doc/content/htgr/httf/lower_plenum_cfd.md

```diff
@@ -0,0 +1,129 @@
+|  Rakes and all other walls   | Adiabatic | N/A  |
+
+The mean LP pressure is 100.5 $kPa$, with which the helium gas has a density of 0.1004 $kg/m^3$. A non-dimensionalization process is performed for the Nek5000 calculations. The reference velocity is 3.47 $m/s$ that is the highest inlet velocity observed for Ring 3 inlet channels. The temperature difference with respect to the side reflector wall temperature (452.37 $K$) is normalized by $\Delta T = 562.20-452.37 = 109.83 (K)$, where the maximum temperature is from Ring 1 inlet channels. During the post-processing, the CFD results can be easily converted back to dimensional quantities for further analyses. 
+
+## Nek5000 Modelling
+
+The reader can find the Nek5000 case files and the associated HTTF lower plenum files (through github LFS) in VTB repository. The mesh information is contained in two files: +httf.re2+ (grid coordiantes, sideset ids, etc.) and +httf.co2+ (mesh cell connectivity). 
+As for the Nek5000 case files, there are +SIZE+, +httf.par+, +httf.usr+, and two auxiliary files +limits.f+ and +utilities.f+ that hosts various post-processing functions, such as solution monitoring, planar avearging, and so forth. 
++SIZE+ is the metafile to specify primary discretization parameters. 
++httf.usr+ is the script where users specify the boundary conditions, turbulence modeling approach and post-processing. 
++httf.par+ is to provide the simulation input parameters, such as material properties, time step size, Reynolds number. 
+
+Now let's dive into the most important case file +httf.usr+. The sideset ids contained in the mesh file are first translated into CFD boundary condition settings in +usrdat2+ block
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat2() end=subroutine usrdat3() include-end=False
+
+The $k-\tau$ URANS model is specified in the following code block
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat3() end=subroutine turb_in(wd,tke,tau) include-end=False
+
+The specific inlet temperature and mass flow rates are implemented in +userbc+ with the non-dimensionalized values
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=if(id_face.eq.3) end=endif include-end=True
+
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
+A characteristics based time-stepping has been used in the simulations to avoid the limitations imposed by the Courant-Friedrichs-Lewy
+(CFL) number due to the explicit treatment of the non-linear convection
+term. This treatment is necessary due to the existence of small local mesh cells created during the unstructured meshing and tet-to-hex conversion. 
+A maximum CFL number of 1.75 has been reached which corresponds to an average time step size of $2.5\times 10^{-5}$.
+A total simulation time of 5.6 is achieved. 
```

Comment:
5.6 hours?

Reply:
maybe just say (non dimensional)

but really would make sense to give time in SI